src/Crypto/WebAuthn/Operation/Registration.hs

```diff
@@ -305,11 +306,50 @@ verifyRegistrationResponse ::
   -- Or () in case of a result.
   Validation (NonEmpty RegistrationError) RegistrationResult
 verifyRegistrationResponse
+  origins
+  rpIdHash
+  registry
+  currentTime
+  options
+  credential =
+      verifyRegistrationResponse'
+        origins
+        rpIdHash
+        registry
+        currentTime
+        options
+        M.CredentialMediationRequirementOptional
```

Comment:
Sets `CredentialMediationRequirementOptional` by default, backwards-compatible change for existing callers.

Internals of the function get moved into `verifyRegistrationResponse'` below